Content.Server/Botany/Systems/MutationSystem.cs

```diff
@@ -200,6 +214,63 @@ private void MutateHarvestType(ref HarvestType val, int bits, int totalbits, flo
             val = HarvestType.SelfHarvest;
     }
 
+    private void MutateGases(ref Dictionary<Shared.Atmos.Gas, float> gases, float min, float max, int bits, int totalbits, float mult)
+    {
+        float p = mult * bits / totalbits;
+        p = Math.Clamp(p, 0, 1);
+        if (!Random(p))
+            return;
+
+        List<Shared.Atmos.Gas> newGas = Enum.GetValues(typeof(Shared.Atmos.Gas)).Cast<Shared.Atmos.Gas>().ToList();
+        var rng = IoCManager.Resolve<IRobustRandom>();
+        float amount = _robustRandom.NextFloat(min, max);
+        Shared.Atmos.Gas gas = rng.Pick(newGas);
+        if (gases.ContainsKey(gas))
+        {
+            gases[gas] += amount;
+        }
+        else
+        {
+            gases.Add(gas, amount);
+        }
+
+    }
+
+    private void MutateChemicals(ref Dictionary<string, SeedChemQuantity> chemicals, int max, int bits, int totalbits, float mult)
+    {
+        float p = mult * bits / totalbits;
```

Comment:
Should be named descriptively

Reply:
If you mean the line 241, that is same piece of code as the previous author of the file used for other 'Mutate' methods. I either re-write them all to be more descriptive or keep the same to have consistancy/readability.

Reply:
See above on *p*.